lib/langchain/utils/token_length/openai_validator.rb

```diff
@@ -28,10 +28,11 @@ class OpenAIValidator < BaseValidator
           "text-embedding-3-large" => 8191,
           "text-embedding-3-small" => 8191,
           "text-embedding-ada-002" => 8191,
-          "gpt-3.5-turbo" => 4096,
+          "gpt-3.5-turbo" => 16385,
```

Comment:
:memo:
gpt-3.5-turbo is now aliased to gpt-3.5-turbo-0125, so I aligned it there.